crates/wash-lib/src/cli/claims.rs

```diff
@@ -798,6 +804,7 @@ mod test {
                 assert!(insecure);
                 assert!(jwt_only);
                 assert!(no_cache);
+                assert!(!wit);
```

Comment:
Nit: this was confusing at first. Consider `assert_eq(wit == false)`?

Reply:
We do the same thing a few lines up with
```
        assert!(!allow_latest);
        assert!(!insecure);
```
so I'd prefer to just be consistent, either way. I prefer this but not to an extreme degree

Reply:
Fair enough